src/UICrafter.Client/Components/AppViewDesigner.razor

```diff
@@ -89,4 +91,15 @@
 
         component.Item.DropZoneID = component.DropzoneIdentifier;
     }
+
+    private async Task Preview()
```

Comment:
I am more a fan of doing preview as a tab at the top with the editor but this is fine for now.